examples/protection_policies_v2/VG_as_protected_resource_v2.yml

```diff
@@ -0,0 +1,405 @@
+# Pre-requisites:
+# 1. Ensure that you have cluster (PE) which is not registered to any PC.
+# 2. Register that cluster with your PC. (Now you have two clusters registered with PC)
+# 3. Modify the firewall rules for the two clusters that are registered with PC using the following commands:
+# SSH into the first cluster (PE1) and run the following command:
+# /usr/local/nutanix/cluster/bin/modify_firewall -f -r <PE2_IP>,<PE2_VIP> -p port1,port2,port3,port4,port5 -i eth0
+# SSH into the second cluster (PE2) and run the following command:
+# /usr/local/nutanix/cluster/bin/modify_firewall -f -r <PE1_IP>,<PE1_VIP> -p port1,port2,port3,port4,port5 -i eth0
+
+# Summary:
+# This playbook will do the following:
+# 1. Create two categories.
+# 2. Create two protection policies with auto retention and synchronous replication configurations.
+# 3. Create two volume groups and associate them with the categories.
+# 4. Promote the first volume group and restore the second volume group.
+# 5. Disassociate the categories from the volume groups.
+# 6. Delete the volume groups.
+# 7. Delete the protection policies.
+# 8. Delete the categories.
+
+- name: Protection policies playbook
+  hosts: localhost
+  gather_facts: false
+  module_defaults:
+    group/nutanix.ncp.ntnx:
+      nutanix_host: <pc_ip>
+      nutanix_username: <user>
+      nutanix_password: <pass>
+      validate_certs: false
+  tasks:
+    - name: Generate random names and set variables
+      ansible.builtin.set_fact:
+        cluster_ext_id: "00062f93-af62-1b48-0374-7a4d34c109e2"
+        random_name: "{{query('community.general.random_string',numbers=false, special=false,length=12)[0]}}"
+        label1: "{{query('community.general.random_string',numbers=false, special=false,length=12)[0]}}"
+        label2: "{{query('community.general.random_string',numbers=false, special=false,length=12)[0]}}"
+        suffix_name: "ansible-category"
+        todelete_categories: []
+        todelete: []
+        cluster:
+          uuid: "00062db4-a450-e685-0fda-cdf9ca935bfd"
+        cluster_availability_zone:
+          uuid: "00062df3-0a98-93d3-0654-e452ba9c0c25"
+        availability_zone_pc_ip: "10.2.0.0"
+        availability_zone_pc_uuid: "bd32fb09-8005-4655-a3a8-086b8ec1b1ea"
+        domain_manager_ext_id: "1e9a1996-50e2-485f-a67c-22355cb43055"
+
+    - name: Setting Variables
```

Comment:
Please set fields directly in task rather than setting them in variables for more straightforward and clear example